deploy.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+cd `dirname ${0}`
+
+print_usage(){
+    echo "    ./deploy.sh [options] <version> <stack_name> [stack-version]"
+    echo ""
+    echo "    Options:"
+    echo "      -h          show help and exit"
+    echo "      --dry-run   do not really deploy"
+    echo "      --live      deploy into production environment (\"aruha\" instead of \"aruha-test\")"
+    echo ""
+}
+
+# default settings
+ACCOUNT="aruha-test"
+DRY_RUN=""
+
+while [[ "x$1" != "x" ]] && [[ ${1:0:1} == "-" ]]; do
+    case $1 in
+        --dry-run)
+            DRY_RUN="--dry-run"
+        ;;
+        --live)
+            ACCOUNT="aruha"
+        ;;
+        -h|--help)
+            print_usage
+            exit 0
+        ;;
+    esac
+    shift
+done
+
+if [[ $# -lt 2 ]]; then
+    echo "Error: Missing arguments."
+    print_usage
+    exit 1
+fi
+
+VERSION="${1}"
+STACK_NAME="${2}"
+STACK_VERSION="${3}"
+
+mai login ${ACCOUNT} && pierone login
+if [ $? -ne 0 ]; then
+    echo "Failed to login to account ${ACCOUNT}"
+    exit 2
+fi
+
+if [[ "x$STACK_VERSION" == "x" ]]; then
+    echo "Stack version is not specified, will try to find current"
+    CURVER=`senza status $NAME | grep $NAME | awk '{print $2}' | sort -rn | head -1`
+    echo "Current stack version is ${CURVER}"
+    if [[ "x${CURVER}" == "x" ]]; then
+        STACK_VERSION='1'
+    else
+        let STACK_VERSION=$CURVER+1
+    fi
+    echo "Calculated stack version is ${STACK_VERSION}"
+fi
+
+ACCOUNT_CONFIG="../nakadi-private/config/accounts/${ACCOUNT}"
+if [[ ! -f ${ACCOUNT_CONFIG} ]]; then
+    echo "Account configuration file ${ACCOUNT_CONFIG} is not found"
+    exit 3
+fi
+
+if [[ "x$ACCOUNT" == "xaruha" ]]; then
+    echo "################################################################################"
+    echo "#                                                                              #"
+    echo "# WARNING: You are deploying into the production environment                   #"
+    echo "#                                                                              #"
+    echo "################################################################################"
+    sleep 3
+fi
+
+# Check if stack configuration exists
+CONFIG_FILE="../nakadi-private/config/$STACK_NAME"
```

Comment:
Probably it should be used as parameter, but for now - it just exists, and I've never experienced situation where this must be done from different paths.